### PR TITLE
Minor correction to viewBox so that all text becomes visible.

### PR DESCRIPTION
**Pull request by fabienpe**
***
<a href="https://pr-preview.s3.amazonaws.com/fabienpe/vc-data-model/pull/1213.html" title="Last updated on Jul 27, 2023, 8:12 AM UTC (ab68453)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1213/d84cf2e...fabienpe:ab68453.html" title="Last updated on Jul 27, 2023, 8:12 AM UTC (ab68453)">Diff</a>

### CODE REVIEW

**Review by brentzundel — APPROVED** [review]

File: diagrams/privacy-spectrum.svg
Comment (TallTed) [inline diagrams/privacy-spectrum.svg:1]: I would like all capital letters (except `ID`, `IDs`, and `Personally Identifiable Info) made lower case.

If relatively easy (I haven't looked at the SVG source, yet), center aligning each stack (both each caption within itself, and the top-to-bottom) except for the `more/less privacy` in the color bar, would substantially improve appearance.

Comment (fabienpe) [reply diagrams/privacy-spectrum.svg:1]: Something like this you mean?
privacy-spectrum

Comment (fabienpe) [reply diagrams/privacy-spectrum.svg:1]: Easy fix. Regarding data correlation via collusion or not, I suspect you might be right, that collusion is not the only path. But I leave it to you and the other editors of this document to decide. My initial intention was only to correct errors I had spotted while reading the document.
privacy-spectrum

**Review by msporny — APPROVED** [review]

**Review by TallTed — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

Thank you for the contribution, @fabienpe ! :)

**Pull request merged**